### ISSUE TITLE
Setting Form Field Value to Zero Not Working

### ISSUE THREAD

**Issue by sky-gregory** [gunjandatta/gd-bs#4]
When attempting to set loading values for a Form component, using a zero results in an empty text field

```javascript
let _form = Components.Form({
   el: targetElement,
   rows,
   //Set default values
   value: {
      "fieldValue": "0"
   }
});
```

**Comment by gunjandatta** [comment]
I can't seem to reproduce this issue. This is the code I used:

```
            var form = GD.Components.Form({
                el: document.getElementById("dev"),
                value: {
                    "fieldValue": "0"
                },
                rows: [
                    {
                        control: {
                            label: "Issue 4",
                            name: "fieldValue",
                            type: GD.Components.FormControlTypes.TextField
                        }
                    }
                ]
            });
```

**Comment by gunjandatta** [comment]
If you have the value as 0 (the number) it won't display. I'll update the code to put the value in.

**Comment by gunjandatta** [comment]
Fixed in gd-bs v1.2.2
